examples/todo-mvc-mem/src/App.tsx

```diff
@@ -148,7 +148,8 @@ function Footer({
   );
 }
 
-export default function App({ list }: { list: TodoList }) {
+export default function App() {
+  const list = useQueryOne(TodoList.queryAll, [], { key: 'list' });
```

Comment:
what's your thoughts on putting `deps` into the options?

e.g.,
```
useQueryOne(TodoList.queryAll, { key: 'list', deps: [] });
```

Reply:
Less idiomatic maybe. I think if we can make `key` generated we may not even need deps? Because if you pass in a different query creator but the LiveResult it generates is the same, it will already use the cached one. If you change some param so that it computes a different result, it will cache miss and evaluate again. But I'm a little fuzzy on if that really makes sense right now.

I guess stuffing it in the object could be fine, and then maybe it disappears someday.